com.unity.uiwidgets/Runtime/rendering/binding.cs

```diff
@@ -76,6 +76,7 @@ public class RendererBinding : SchedulerBinding {
         }
 
         void _handlePersistentFrameCallback(TimeSpan timeStamp) {
+            window.updateSafeArea();
```

Comment:
why call it here? is there any better place to call this (e.g., inside the callback of ViewportMetricsChanged )